freqtrade/freqtradebot.py

```diff
@@ -255,6 +255,24 @@ def get_target_bid(self, ticker: Dict[str, float]) -> float:
         balance = self.config['bid_strategy']['ask_last_balance']
         return ticker['ask'] + balance * (ticker['last'] - ticker['ask'])
 
+    def _get_trade_stake_amount(self) -> float:
+        stake_amount = self.config['stake_amount']
+        avaliable_amount = exchange.get_balance(self.config['stake_currency'])
+
+        if stake_amount == 'unlimited':
```

Comment:
I think `'unlimited'`should be extracted to `constants.py` and converted into a constant. This will avoid eventual bugs via typo in the future. don't see a bug in this PR - but assuming this is merged and built upon i would rather not use a hardcoded string

Reply:
Totally agree! Will fix it.